release/release.go

```diff
@@ -108,3 +64,26 @@ func ReadLsb() (*Lsb, error) {
 
 	return lsb, nil
 }
+
+var dpkgStatusPath = "/var/lib/dpkg/status"
+
+// OnClassic returns whether the process is running inside a
+// classic Ubuntu system or a native Ubuntu Core image.
+func OnClassic() bool {
```

Comment:
Actually, I take that back. It's actually even simpler if we cache it. Let me give it a try.